monitoring/grafana/dashboards/generate-jsonnet.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+BASEDIR=$(dirname "$0")
+
+for file in $BASEDIR/jsonnet/*; do
+    name=$(basename $file)
+    jsonnet $file > $BASEDIR/${name%.jsonnet}.json
+done
```

Comment:
It seems jsonnet supports generating multiple files from the single jsonnet file. Have you considered that? Is it worthy?

Reply:
Thanks @epuertat . I have updated the PR with generating multiple json files from single jsonnet file. PTAL.